src/Xamarin.Android.Build.Tasks/Utilities/TypeMapGenerator.cs

```diff
@@ -53,18 +54,39 @@ internal sealed class TypeMapEntry
 			public int ModuleIndex = -1;
 		}
 
-		internal sealed class ModuleData
+		internal sealed class ModuleReleaseData
 		{
 			public Guid Mvid;
 			public byte[] MvidBytes;
 			public AssemblyDefinition Assembly;
-			public TypeMapEntry[] Types;
-			public Dictionary<uint, TypeMapEntry> DuplicateTypes;
+			public TypeMapReleaseEntry[] Types;
+			public Dictionary<uint, TypeMapReleaseEntry> DuplicateTypes;
 			public string AssemblyName;
 			public string AssemblyNameLabel;
 			public string OutputFilePath;
 
-			public Dictionary<string, TypeMapEntry> TypesScratch;
+			public Dictionary<string, TypeMapReleaseEntry> TypesScratch;
+		}
+
+		internal sealed class TypeMapDebugEntry
+		{
+			public string JavaName;
+			public string JavaLabel;
+			public string ManagedName;
+			public string ManagedLabel;
+			public int JavaIndex;
+			public int ManagedIndex;
+		}
+
+		// Widths include the terminating nul character nor the padding!
```

Comment:
Should "nor" be "not" here?  Or should it read "Widths include *neither* the terminating nul character nor the padding"?